presto-proxy/pom.xml

```diff
@@ -154,16 +154,15 @@
             <scope>runtime</scope>
         </dependency>
 
-        <!-- for testing -->
         <dependency>
-            <groupId>org.testng</groupId>
-            <artifactId>testng</artifactId>
-            <scope>test</scope>
+            <groupId>com.facebook.presto</groupId>
```

Comment:
We shouldn't import `presto-main` in `presto-proxy`.

@mayankgarg1990 do we want to enforce security in `presto-proxy`?

Reply:
Yes, we need to have authn and authz in every single FB endpoint

Reply:
The `presto-proxy` is a standalone application. It is not defining any endpoints used by Presto. 

From `presto-proxy` readme

```
This proxy server allows clients to securely access a remote Presto
server (or set of servers) without having direct access to the server.
For example, if the Presto server is behind a firewall, only the proxy
needs to be exposed to the client.
```

also

```
If so configured, the proxy will generate JWT access tokens containing
the principal from the TLS client certificate presented to the proxy.
This allows using the proxy in secure environments without needing
the proxy to perform explicit validation of the username and principal.
```

https://github.com/prestodb/presto/blob/master/presto-proxy/README.txt

It feels like the Proxy shouldn't enforce authorization. Instead it should be enforced by the presto server itself.

Reply:
Makes sense :) We can skip this then.